apple/repositories.bzl

```diff
@@ -127,10 +127,23 @@ def apple_rules_dependencies(ignore_version_differences = False):
     )
 
     _maybe(
-        http_file,
+        http_archive,
+        name = "subpar",
+        urls = [
+            "https://github.com/google/subpar/archive/2.0.0.tar.gz",
+        ],
+        strip_prefix = "subpar-2.0.0",
+        sha256 = "b80297a1b8d38027a86836dbadc22f55dc3ecad56728175381aa6330705ac10f",
+        ignore_version_differences = ignore_version_differences,
+    )
+
+    _maybe(
+        http_archive,
         name = "xctestrunner",
-        executable = 1,
-        sha256 = "298846d5ad7607eba33e786149c2b642ffe39508d4a99468a8280871d902fe5d",
-        urls = ["https://github.com/google/xctestrunner/releases/download/0.2.14/ios_test_runner.par"],
+        urls = [
+            "https://github.com/segiddins/xctestrunner/archive/segiddins/allow-running-ios-test-runner-external-repo.tar.gz",
+        ],
+        strip_prefix = "xctestrunner-segiddins-allow-running-ios-test-runner-external-repo",
+        sha256 = "91097584d3cf94fd0da1192f7a49389e65df0200144705b59af4fa8717762b8a",
```

Comment:
Looks like this got committed by accident

Reply:
Not really -- needed to point to my fork of xctestrunner in order to test out the changes